shared/message/support/GlobalConfig.proto

```diff
@@ -20,8 +20,9 @@ syntax = "proto3";
 
 package message.support;
 
-// Field dimensions as defined in the Robocup rules:
 message GlobalConfig {
+    /// The robots number in the team. Starts at 1. Should be the same as the robots number (nugus1, nugus2, etc)
     uint32 player_id = 1;
-    uint32 team_id   = 2;
+    /// Unique team number, assigned by the league
```

Comment:
Not sure if we want to link this here or in NUbook, but we find out our team ID from https://github.com/RoboCup-Humanoid-TC/GameController/tree/master/resources/config

Reply:
I think it would be good to have that detail here (nubook could use it as well). Could you please elaborate on how we get it from there @ysims?

Reply:
Go to the folder for our league (eg this year was `hl_sim_kid` (humanoid league simulation kid size)), go to `teams.cfg`, then just find `<number>=NUbots` and the `<number>` is our team ID.